docs/whatsnew/7.0.rst

```diff
@@ -84,18 +84,35 @@ The minimum required versions of the following packages have been updated:
 - pandas>=2.1.0
 - aiobotocore>=2.6.0
 
-Maps now plot with autoalignment by default
-===========================================
+Polarimeter to Unify the Corona and Heliosphere (PUNCH) map source
+==================================================================
 
-The "autoalignment" functionality of the `~sunpy.map.Map` method :meth:`~sunpy.map.GenericMap.plot` has been significantly enhanced and is now enabled by default.
+There is now support for PUNCH level 1 products within sunpy.
+
+.. image:: 7.0-punch.png
+   :alt: A plot from sunpy of a PUNCH level 1 file.
+
+Additional new map sources
```

Comment:
Why do we have this heading and also PUNCH above? Feels like we should be giving them all pretty equal billing.

Reply:
But I will fix.

Reply:
Fixed.